### ISSUE TITLE
Field Names from FileMakerPro

### ISSUE THREAD

**Issue by jimrandell** [wise-coders/dbschema#39]
- DbSchema 8.4.1 build 210323
 - Microsoft Windows [Version 10.0.19041.867]
 - FileMakerPro 18 
 - FileMaker JDBC drivers: v16 (the drivers you provide), v18, and v19

Hello and thank you for your help. I really like your product, and will enthusiastically purchase a license if I can resolve this blocker..

Using the Relational Data Explorer fails to display data, but it does returns a SQL Syntax error. 
Viewing the SQL History, the issue becomes clear to me: the SQL is failing due to two field names. 
The names of the troublesome fields are:
 - Secure
 -  _acctNum
 
I can easily correct for this by pasting the failed SQL from the SQL History, into a new SQL Editor, and wrapping the field names in quotes, like this:
 - "Secure"
 -  "_acctNum"

...but with dozens of tables, many with HUNDREDS of fields, I need a way for such fields to have quotes added automatically (exactly as happens with field names containing spaces. for example, in the SQL History, I see a field called: first name, automatically resolves to "first name").

I need a place to add "Rules for Auto-Quoting". You already have at least one such rule: 
 - "field names containing spaces". 

I need to add: 
 - "Field names starting with underscore"
 -  "Field name is Secure"

I suspect this already exists, but I cant find it. can you point me in the right direction, and perhaps give me some example syntax, in case i'm a bit too dim to figure it out myself?

Thank you so much for your assistance!!!!

**Comment by dprutean** [comment]
Thank you for reporting this issue. We will fix this right this week. A beta update will be available next Monday on https://dbschema.com/beta.php

**Comment by jimrandell** [comment]
Thank you guys!
Been thinking about this....Filemaker allows some pretty funky character/strings to be used in field names, requiring developers to wrap double quotes around such field names when used in SQL.

some thoughts from someone familiar with FileMaker's idiosyncrasies:
 - _**perhaps the simplest solution**_, when using the FM driver, to wrap ALL field names in double quotes
 - another option, a checkbox called: "Wrap All Field Names In Double Quotes"
 - my original thought might be needlessly complex...creating an interface where users can build "Rules For Quoting Field Names"
 - another option is a checkbox next to each field name in the Schema Tree Browser to "Wrap Field Name in Double Quotes"

...thank you for your help with this!!!

**Comment by jimrandell** [comment]
I have just downloaded and installed 21043 and I see you have corrected one of the two issues I found:
 -  Field names which start with an underscore are now wrapped in double quotes.
 - BUT the field named Secure **IS STILL NOT QUOTED**, and causes the sql to fail. 

this must be an illegal word. I'm not sure what other words may be illegal, and still suggest an option to wrap all field names in quotes.
